src/components/Cataloguing/Marc/Record/index.js

```diff
@@ -457,22 +484,26 @@ class Record extends React.Component<
   };
 
   render() {
-    const { leaderData } = this.props;
+    const { leaderData, translate, data: { search: { segment } } } = this.props;
     const { isEditMode, id } = this.state;
     const bibliographicRecord = this.getCurrentRecord();
 
+    const childProps = { ...this.props };
+    delete childProps.emptyRecord;
+    delete childProps.emptyRecordAuth;
+
+    const paneTitle = segment === C.SEARCH_SEGMENT.BIBLIOGRAPHIC
```

Comment:
This should be simplified. You could just test for edit mode, then have other conditions in an ELSE clause for example.

Reply:
Now, first I ask about the edit mode and then for other conditions in an else